core/unit/id.go

```diff
@@ -23,6 +23,16 @@ func NewUUID() (UUID, error) {
 	return UUID(id.String()), nil
 }
 
+// MustNewUUID is a convince function for generating a new unit uuid
+// and panics on error. Only used in tests.
+func MustNewUUID() UUID {
+	uuid, err := NewUUID()
+	if err != nil {
+		panic(err)
+	}
+	return uuid
+}
```

Comment:
This should be in `core/unit/testing` see the existing pattern for charms and applications.

Reply:
yeah, totally forgot that